snowflake/datadog_checks/snowflake/queries.py

```diff
@@ -191,12 +191,16 @@
 PipeHistory = {
     'name': 'pipe.metrics',
     'query': (
-        'select pipe_name, avg(credits_used), sum(credits_used), avg(bytes_inserted), sum(bytes_inserted), '
-        'avg(files_inserted), sum(files_inserted) from pipe_usage_history '
-        'where start_time >= DATEADD(hour, -24, current_timestamp()) group by 1;'
+        'select history.pipe_name,p.pipe_schema, p.pipe_catalog, '
+        'avg(credits_used), sum(credits_used), avg(bytes_inserted), sum(bytes_inserted), '
+        'avg(files_inserted), sum(files_inserted) from pipe_usage_history as history '
+        'join snowflake.account_usage.pipes p on p.pipe_id = history.pipe_id '
+        'where start_time >= DATEADD(hour, -24, current_timestamp()) group by 1,2,3;'
     ),
     'columns': [
         {'name': 'pipe', 'type': 'tag'},
+        {'name': 'pipe_schema', 'type': 'tag'},
+        {'name': 'pipe_catalog', 'type': 'tag'},
```

Comment:
Assuming that this do refer to these concepts, maybe it would make sense to use the same tags that are used for other metrics? That should make it easier to cross-reference.

```suggestion
        {'name': 'schema', 'type': 'tag'},
        {'name': 'database', 'type': 'tag'},
```